### PR TITLE
Adds Show and Tell docs

### PR DESCRIPTION
**Pull request by ashfurrow**
Hey @jonallured – we had a really good frontend practice meeting today, good because we didn't discuss our usual stuff but instead just saw a few presentations of work-in-progress stuff from @anandaroop and @orta. We were talking afterwards and really liked the format, so I've created a weekly event to try these out. Roop mentioned you were behind a similar event on your team and I wanted to get your thoughts on this plan. What do you think?

### CODE REVIEW

**Comment by jonallured** [comment]
Wow, so excited about this!! How can I help and squee!!

**Comment by ashfurrow** [comment]
Cool! If you have any feedback on the proposed format, please let me know. Otherwise, maybe you could present the idea on Monday's standup and ask for volunteers? It would help if it weren't always Orta and I asking for volunteers 😄

**Comment by anandaroop** [comment]
Yay! One approach might be to not even ask for volunteers ahead of time (or at least not try to fill up a whole session ahead of time). This allows for more of an open-mic style, which usually worked well for us on the old team. There is the risk that no one will have anything but that can be weighed against the advantage of lowering the barrier for entry and keeping things more informal and chill.

**Comment by orta** [comment]
I think that's a good idea, maybe we prime people a day before, and an hour before that they can think of things to show - and just have the S&T on the announcements section each time

**Comment by ashfurrow** [comment]
Cool, let's try that format out. I'll update the docs.

File: meta/show_and_tell.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Show & Tell
+
+Show & Tell is a new weekly event we're trying out to get more visibility into what our teammates have been working on recently. It is scheduled weekly for a half hour each week (see the shared Engineering Open Meetings calendar) and consists of three ten-minute walkthroughs. These walkthroughs are a presentation of works-in-progress and *no* preparation is expected of these presenters. We get people exciting about Show & Tell at [Monday standups](open_standup.md) and then do an *@here* an hour beforehand to remind people. We then meet and, in an open mic-style, developers step forward to show and tell!
+
```
Comment (anandaroop) [inline meta/show_and_tell.md]: #minor "exciting" &rarr; "excited"

Thanks @ashfurrow !

**Comment by jonallured** [comment]
This is awesome and I would be happy to talk about it, including priming people at Monday standup.

**Pull request merged**